addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -140,13 +145,19 @@ var ActivityMenu = Widget.extend({
         // Necessary because activity_ids of mail.activity.mixin has auto_join
         // So, duplicates are faking the count and "Load more" doesn't show up
         context['force_search_count'] = 1;
+        
+        var domain = [['activity_ids.user_id', '=', session.uid]]
+        if (data.domain != null) {
+            domain.unshift('&', data.domain)
```

Comment:
Same remark as above applies here, just don't forget to update :) .

Reply:
Updated!